inference-engine/ie_bridges/java/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 Intel Corporation
+
+cmake_minimum_required(VERSION 3.0.0)
+
+find_package(Java REQUIRED)
+include(UseJava)
```

Comment:
we need `README.md` into `inference-engine/ie_bridges/java` with details what is java for OpenVINO and how to build it.